addons/hw_drivers/views/pos_display.html

```diff
@@ -72,30 +82,26 @@
             }
             .pos-display-box {
                 padding: 10px 20px;
-                background: rgba(0, 0, 0, 0.17);
-                border: 1px solid rgba(0, 0, 0, 0.06);
-                box-shadow: 1px 1px 0px 0px rgba(60, 60, 60, 0.4);
-                color: #fff;
+                background: rgba(255, 255, 255, 0.17);
+                border: 1px solid rgba(255, 255, 255, 0.06);
+                box-shadow: 0 0 5px 0 rgba(60, 60, 60, 0.4);
                 border-radius: 8px;
                 width: 500px;
                 margin-top: 20px;
             }
-            .pos-display-box hr {
-                background-color: #fff;
-            }
             .info-text {
                 font-size: 15px;
             }
-            .table-pos-info {
-                color: #fff;
+            .iot-devices-section {
+                display: none;
```

Comment:
hide "IoT Devices" section on first load (before the first ajax request to get iotbox connected devices)